### PR TITLE
Anonymous Feature overhaul

### LINKED ISSUE #89: Enhancements to Anonymous Mode
(fellow WWU teacher here)

We are using the Overflow-Plug-in with great success in „Analysis I.“ and believe that part of the success is due to the fact, that people can ask questions anonymously. However the activity in that forum is so high, that the discussions become hard to follow if more than two people participate _due_ to the fact, that all users look the same. 

I suggest, that instead of „anonymous user“ it says „questioner“, „answerer 1“, „answerer 2“ and so on. That would make discussions much more easier to follow.
Adding some sort of „questioner“ label would also be helpful for the regular mode of the Overflow plug-in. With that in place one could simply number all participants of a question.

### PR DESCRIPTION
**Pull request by justusdieckmann**
Fixes #89 

Changes:
- Questioner is highlighted in blue, regardless of anonymity mode.
- In "Only Questions anonymous"-mode, the questioner is also anonymous when writing an answer / a comment
- anonymous Users are named "Questioner"/"Answerer #i" instead of "Anonymous"

Screenshot from 2022-03-09 17-07-13

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -512,6 +506,10 @@ span.unread {
     flex-grow: 1;
 }
 
+.moodleoverflowpost .questioner {
+    background-color: rgba(0, 127, 255, 0.1);
```
Comment (Laur0r) [inline styles.css:510]: The highlighting is a good idea but can you make it a bit more subtle? I feel like the color is a bit too strong so that one automatically only focuses on the name and not the content.

**Review by Laur0r — CHANGES_REQUESTED** [review]
Great work!

File: styles.css (hunk 1 above)
Comment (justusdieckmann) [reply styles.css]: I made the color a bit lighter :)

**Comment by Laur0r** [comment]
Looks great, thanks!
